napari/_vispy/_tests/test_vispy_camera.py

```diff
@@ -185,12 +184,9 @@ def test_camera_orientation_2d(make_napari_viewer):
 
 @pytest.mark.xfail(
```

Comment:
```suggestion
@pytest.mark.skipif(
```
Skip seems like the proper strategy here until we resolve the issue.
We don't expect this test to fail, it just does?